components/ai_chat/resources/page/components/conversation_list/style.module.scss

```diff
@@ -4,35 +4,28 @@
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
 .list {
-  padding: 16px;
+  // padding: 16px;
 }
 
-.textBase {
-  p {
-   margin: 0;
-   white-space: pre-wrap;
-  }
- }
+.turnBase {
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  font: var(--leo-font-secondary-default-regular);
+  color: var(--leo-color-text-primary);
+  padding: 16px;
+}
 
 .turnAI {
-  @extend .textBase;
-  padding: 16px;
+  @extend .turnBase;
```

Comment:
IMO let's try not to use sass features, and stick to css. Most styles in brave-core are styled-components. I think we can allow css in lieu of a decision to go forward in a single direction. But adding sass-specific features I think adds yet another framework which not everyone knows the nuances of. OK to keep .scss ending so we get compile-type support for variables which we may start using from the Design System instead of css variables so that we get compiler errors when they change

Reply:
makes sense. done.